apps/web/src/components/AssessmentStepTracker/AssessmentResults.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import BookmarkIcon from "@heroicons/react/24/outline/BookmarkIcon";
+
+import {
+  ArmedForcesStatus,
+  AssessmentResult,
+  AssessmentStepType,
+  Maybe,
+} from "@gc-digital-talent/graphql";
+import { Board, Link } from "@gc-digital-talent/ui";
+
+import { getFullNameLabel } from "~/utils/nameUtils";
+
+import useRoutes from "../../hooks/useRoutes";
+import { getDecisionInfo, sortResults } from "./utils";
+
+interface PriorityProps {
+  type: "veteran" | "entitlement";
+}
+
+const Priority = ({ type }: PriorityProps) => {
+  const intl = useIntl();
+
+  return (
+    <span
+      data-h2-align-items="base(center)"
+      data-h2-display="base(inline-flex)"
+      data-h2-background-color="base(black) base:dark(white)"
+      data-h2-color="base(white) base:dark(black)"
+      data-h2-padding="base(0 x.125)"
+      data-h2-radius="base(rounded)"
+      data-h2-font-weight="base(700)"
+      data-h2-transition="base(all .2s ease)"
+    >
+      <span data-h2-line-height="base(1)" data-h2-font-size="base(0.6rem)">
+        {type === "veteran"
+          ? intl.formatMessage({
+              defaultMessage: "VA",
+              id: "KXy+Ei",
+              description: "Short form code representing 'veteran'",
+            })
+          : intl.formatMessage({
+              defaultMessage: "P",
```

Comment:
Is P an official Government of Canada short forms? One letter short forms can be very confusing since so many words can begin with a single letter, or, in this instance, the letter P.